automated install/basic-install.sh

```diff
@@ -2442,15 +2442,15 @@ FTLcheckUpdate() {
         if [[ ${ftlLoc} ]]; then
             local FTLversion
             FTLversion=$(/usr/bin/pihole-FTL tag)
+
             local FTLlatesttag
+            FTLlatesttag=$(curl -s https://api.github.com/repos/pi-hole/FTL/releases/latest | jq -r .tag_name)
 
-            if ! FTLlatesttag=$(curl -sI https://github.com/pi-hole/FTL/releases/latest | grep --color=never -i Location: | awk -F / '{print $NF}' | tr -d '[:cntrl:]'); then
+            if [ -z "${FTLlatesttag}" ]; then
```

Comment:
This will fail if there is a bad response from the `curl` command.
```sh
Viking:~$ tag=$(curl -s https://api.github.com/repos/pi-hole/FTL/releases/lates | jq -r .tag_name)
Viking:~$ echo $tag
null
```

Reply:
I tested and never got `null`, but I will replace the command with:

```
curl -s https://api.github.com/repos/pi-hole/FTL/releases/lates | jq -r '.tag_name | values'
```

Reply:
Do you want to use `jq` for the test, like this?

```
curl -s https://api.github.com/repos/pi-hole/FTL/releases/lat | jq '.tag_name == "${FTLversion}"'
```

Reply:
The current code will work for any `curl` response:
- if `curl` returns a valid JSON, `jq` will try to find and return `tag_name`. 
- if no `tag_name` is found in JSON response, an empty string will be returned.
- if `curl` responds with anything that is not a valid JSON, `jq` command will return an empty string.

Command details:
```console
jq -sRr 'fromjson? | .tag_name | values'
```

- `-s` will read the entire input file into a large array before apply the filters;
- `-R` will not parse the input as JSON. When combined with `-s`, it reads the whole response into a single long string;
- The `?` is the error suppression operator and works like a simple `try/catch`.
- `fromjson?` will **try** to parse the text into JSON format. If not possible will return `null`. 
- `.tag_name` will return the value of "tag_name" (or `null` if not found);
- `values` will filter the result and return only "non null values" (returning an empty string in case of `null`).
- `-r` will output the result as plain text (without the quotes).